aspnetcore/fundamentals/servers/yarp/toc.yml

```diff
@@ -0,0 +1,74 @@
+- name: Getting Started
+  uid: fundamentals/servers/yarp/getting-started
+- name: Configuration
+  items:
+  - name: Configuration Files
+    uid: fundamentals/servers/yarp/config-files
+  - name: Configuration Providers
+    uid: fundamentals/servers/yarp/config-providers
+  - name: Configuration Filters
+    uid: fundamentals/servers/yarp/config-filters
+  - name: HTTP client configuration
+    uid: fundamentals/servers/yarp/http-client-config
+- name: Proxy Features
+  items:
+    - name: Header Routing
+      uid: fundamentals/servers/yarp/header-routing
+    - name: Query Parameter Routing
+      uid: fundamentals/servers/yarp/queryparameter-routing
+    - name: Transforms
+      uid: fundamentals/servers/yarp/transforms
+    - name: Load Balancing
+      uid: fundamentals/servers/yarp/load-balancing
+    - name: Session Affinity
+      uid: fundamentals/servers/yarp/session-affinity
+    - name: Destinations Health Checks
+      uid: fundamentals/servers/yarp/dests-health-checks
+    - name: Rate Limiting
+      uid: fundamentals/servers/yarp/rate-limiting
+    - name: Timeouts
+      uid: fundamentals/servers/yarp/timeouts
+    - name: Authentication and Authorization
+      uid: fundamentals/servers/yarp/authn-authz
+    - name: Output Caching
+      uid: fundamentals/servers/yarp/output-caching
+    - name: Distributed Tracing
+      uid: fundamentals/servers/yarp/distributed-tracing
+    - name: A/B Testing
+      uid: fundamentals/servers/yarp/ab-testing
+- name: Protocols & Guidance
+  items:
+    - name: Header Guidelines
+      uid: fundamentals/servers/yarp/header-guidelines
+    - name: HTTPS & TLS
+      uid: fundamentals/servers/yarp/https-tls
+    - name: Proxying gRPC
+      uid: fundamentals/servers/yarp/grpc
+    - name: HTTP/3
+      uid: fundamentals/servers/yarp/http3
+    - name: Cross-Origin Requests (CORS)
+      uid: fundamentals/servers/yarp/cors
+    - name: Lets Encrypt
```

Comment:
@samsp-msft  here's you list fixed up into the format we use. When we're ready, this should plug into our root `toc.yml`